src/wasm-builder.h

```diff
@@ -229,6 +230,13 @@ class Builder {
     ret->finalize(type);
     return ret;
   }
+  template<size_t N>
+  Block* makeBlock(const SmallVector<Expression*, N>& items) {
```

Comment:
Would it be worth deduplicating all the different versions of `makeBlock` into a few templatized versions that can take any iterable for the items?

Reply:
#5583